README.md

```diff
@@ -4,5 +4,5 @@ The project uses [Create React App](https://create-react-app.dev/)
 
 ## How to run the project
 
-- Run `yarn install` to install all required project dependencies
-- Run `yarn start` to run project
+- run `yarn install` to install all required project dependencies.
+- run `yarn start` to run project.
```

Comment:
This seems unrelated

Reply:
Do you want me to add a PR just for removing this?